.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@Workiva/skreams
```

Comment:
Based on the OSS guidelines, it sounds like the requirement is that we're required reviewers IFF the github actions themselves change.

I'm on board for just requiring us all the time too, though

Reply:
Question though: does this require a review from us, or just request us as an optional reviewer?